lib/plugins/aws/package/compile/functions/index.test.js

```diff
@@ -2432,5 +2432,56 @@ describe('AwsCompileFunctions', () => {
         ).to.deep.equal(compiledFunction);
       });
     });
+
+    it('should set Condition when specified', () => {
+      awsCompileFunctions.serverless.service.functions = {
+        func: {
+          handler: 'func.function.handler',
+          name: 'new-service-dev-func',
+          condition: 'IsE2eTest',
+        },
+      };
+
+      return expect(awsCompileFunctions.compileFunctions()).to.be.fulfilled.then(() => {
+        expect(
+          awsCompileFunctions.serverless.service.provider.compiledCloudFormationTemplate.Resources
+            .FuncLambdaFunction.Condition
+        ).to.equal('IsE2eTest');
+      });
+    });
+
+    it('should set Condition when specified', () => {
```

Comment:
Isn't this the same test as the one above? 🤔

Reply:
haha, copy-and-paste error :-P will fix it now